apps/studio/e2e/pageActions/footerActions/clickOnCopyToClipboardOption.ts

```diff
@@ -0,0 +1,5 @@
+import { Footer } from "../../pageComponents/Footer";
+
+export const clickOnCopyToClipboardOption = async (footer: Footer, fileType: string): Promise<void> => {
```

Comment:
Sorry if I'm being too nitpicky lol. Are we planning to add more stuff on each footer action file? I'd rather put all footer actions code in the `index.ts` so we could immediately see the code without jumping around files :)